lib/v2/lvv2/news.js

```diff
@@ -0,0 +1,74 @@
+const got = require('@/utils/got');
+const cheerio = require('cheerio');
+const { parseDate } = require('@/utils/parse-date');
+const { art } = require('@/utils/render');
+const path = require('path');
+
+const rootUrl = 'https://lvv2.com';
+
+const titleMap = {
+    'sort-realtime': {
+        't-month': '24小时榜 一月内',
+        't-week': '24小时榜 一周内',
+        't-day': '24小时榜 一天内',
+        't-hour': '24小时榜 一小时内',
+    },
+    'sort-hot': '热门',
+    'sort-new': '最新',
+    'sort-score': {
+        't-month': '得分 一月内',
+        't-week': '得分 一周内',
+        't-day': '得分 一天内',
+        't-hour': '得分 一小时内',
+    },
+};
+
+module.exports = async (ctx) => {
+    const channel = ctx.params.channel;
+    const sort = (channel === 'sort-realtime' || channel === 'sort-score') && !ctx.params.sort ? 't-week' : ctx.params.sort;
+    const url = `${rootUrl}/${channel}/${sort}`;
+
+    const response = await got(url);
+    const $ = cheerio.load(response.data);
+    const list = $('div.spacer > div')
+        .map((_, item) => ({
+            title: $(item).find('h3 > a.title').text().trim(),
+            author: $(item).find('a.author').text().trim(),
+            link: new URL($(item).find('h3.title > a.title').attr('href'), rootUrl).href,
```

Comment:
```xml
<item>
<title>
<![CDATA[ 2月20日深夜，郑州荥阳张女士睡的正香，一名男子突然闯进卧室，企图对她图谋不轨，“他就按着床想要对我做什么”，所幸张女士奋力反抗之下，男子停止了骚扰，她... ]]>
</title>
<description>
<![CDATA[ <a href="https://weibo.com/1708877153/LgYSEyPpB" target="_blank">文章链接</a> ]]>
</description>
<pubDate>Fri, 25 Feb 2022 17:32:32 GMT</pubDate>
<guid isPermaLink="false">https://lvv2.com/t/3654339/title-2%E6%9C%8820%E6%97%A5%E6%B7%B1%E5%A4%9C%EF%BC%8C%E9%83%91%E5%B7%9E%E8%8D%A5%E9%98%B3%E5%BC%A0%E5%A5%B3%E5%A3%AB%E7%9D%A1%E7%9A%84%E6%AD%A3%E9%A6%99%EF%BC%8C%E4%B8%80%E5%90%8D%E7%94%B7%E5%AD%90%E7%AA%81%E7%84%B6%E9%97%AF%E8%BF%9B%E5%8D%A7%E5%AE%A4%EF%BC%8C%E4%BC%81%E5%9B%BE%E5%AF%B9%E5%A5%B9%E5%9B%BE%E8%B0%8B%E4%B8%8D%E8%BD%A8%EF%BC%8C%E2%80%9C%E4%BB%96%E5%B0%B1%E6%8C%89%E7%9D%80%E5%BA%8A%E6%83%B3%E8%A6%81%E5%AF%B9%E6%88%91%E5%81%9A%E4%BB%80%E4%B9%88%E2%80%9D%EF%BC%8C%E6%89%80%E5%B9%B8%E5%BC%A0%E5%A5%B3%E5%A3%AB</guid>
<link>https://lvv2.com/t/3654339/title-2%E6%9C%8820%E6%97%A5%E6%B7%B1%E5%A4%9C%EF%BC%8C%E9%83%91%E5%B7%9E%E8%8D%A5%E9%98%B3%E5%BC%A0%E5%A5%B3%E5%A3%AB%E7%9D%A1%E7%9A%84%E6%AD%A3%E9%A6%99%EF%BC%8C%E4%B8%80%E5%90%8D%E7%94%B7%E5%AD%90%E7%AA%81%E7%84%B6%E9%97%AF%E8%BF%9B%E5%8D%A7%E5%AE%A4%EF%BC%8C%E4%BC%81%E5%9B%BE%E5%AF%B9%E5%A5%B9%E5%9B%BE%E8%B0%8B%E4%B8%8D%E8%BD%A8%EF%BC%8C%E2%80%9C%E4%BB%96%E5%B0%B1%E6%8C%89%E7%9D%80%E5%BA%8A%E6%83%B3%E8%A6%81%E5%AF%B9%E6%88%91%E5%81%9A%E4%BB%80%E4%B9%88%E2%80%9D%EF%BC%8C%E6%89%80%E5%B9%B8%E5%BC%A0%E5%A5%B3%E5%A3%AB</link>
<author>
<![CDATA[ 微博 ]]>
</author>
</item>
```

Some URLs have their title encoded. You can remove it safely (e.g., https://lvv2.com/t/3653893). If the title changes, it may cause duplicated items in some readers as the links is served as guid.

Reply:
OK, thx.